### DISCUSSION TITLE
이벤트 UI의 기능에 대하여

### DISCUSSION THREAD

**Discussion by mille-feu** [Tiny-Metro/Poly-Metro#241] [Q&A]
[이벤트 UI 노션 페이지](https://www.notion.so/3beab9442e6942bfbf2bddad2c4cef1b)를 참조했습니다.

- 화면 중앙 상단의 Event버튼과, 화면 좌측 상단의 종 버튼은 역할이 어떻게 다른가요

**Comment by seong0929** [comment]
- 화면 상단: 뉴스 형식으로 이벤트 관련 모든 것과 등장 시간을 알려줌
- 종 모양의 UI: 플레이어가 확인하지 못한 역 생성을 알려주는 UI
- 종 모양 UI 대신 화면 상단에 나오는 이벤트 팝업창에서 탭을 나눠 역 생성을 알려주는 형식으로 변경(돋보기 기능 삭제)

**Comment by mille-feu** [comment]
[23.07.12회의](https://github.com/Tiny-Metro/Tiny-Metro/wiki/Dev-Diary:-All-in-One-%2323)에서 논의 완료
